sktime/forecasting/scinet.py

```diff
@@ -0,0 +1,286 @@
+"""Deep Learning Forecaster using SCINet Forecaster."""
+
+from sktime.forecasting.base.adapters._pytorch import BaseDeepNetworkPyTorch
+
+
+class SCINetForecaster(BaseDeepNetworkPyTorch):
+    """SCINet-Linear Forecaster.
+
+    Implementation of the SCINet forecaster.
+
+    Core logic is directly copied from the curelab SCINet implementation
+    at github.
```

Comment:
you should include citations, by adding superscript from links in references. take a look at `LTSFLinearForecaster`